ocs_ci/cleanup/aws/cleanup.py

```diff
@@ -131,28 +131,46 @@ def aws_cleanup():
         required=False,
         help="The name of the AWS region to delete the resources from"
     )
+    parser.add_argument(
+        '--force',
+        action='store_true',
+        required=False,
+        help="Skip confirmation prompt"
```

Comment:
Maybe worth to mention also about destructive operation.

Reply:
Done